portability-types-client/src/main/java/org/dataportabilityproject/types/client/transfer/SimpleLoginRequest.java

```diff
@@ -0,0 +1,28 @@
+package org.dataportabilityproject.types.client.transfer;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+public class SimpleLoginRequest {
+  private final String username;
+  private final String password;
```

Comment:
TODO: Determine how to avoid storing the password in the clear.

Reply:
Done. Note that this is also only used in the microsoft mail auth flow. We can delete it once we move that over to oauth or Live Sync.